### PR TITLE
BS Fullhelm minor adjustments

### PR DESCRIPTION
**Pull request by Kaksisilma**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

### Sprite Changes

* Very minor alterations to the sprites of Blackshield's Fullhelm in order to make them pair better with plate carriers, especially in the tan and green variants. Hooded versions were not altered.

* Added an Anthropomorphic style variant to the Fullhelm, shown below.

<details>
<summary>
       Here's how the grey variant looks like:
</summary>
OLD:

Helmold

NEW:

Helmnew

</details>

<details>
<summary>
       Anthropomorphic Full Helmet
</summary>

Anthro
</details>

<hr>

### Code Changes

* Full helmets now have the HIDEFACE flag, including the Ablative BS helmet.
* The normal, non-full helmet variant is now again the standard helmet variant in BS vouchers.
* Normal helmets can be upgraded to full variants through the crafting menu, requiring two sheets of steel, two sheets of glass, duct tape, a welder and a helmet.

<hr>

</details>

## Changelog
:cl:
imageadd: Anthro alternative style for the BS full helmet. Minor sprite changes to the alternative styles to make them fit better with the plate carrier and uniform color schemes. Hooded variants were untouched.
tweak: BS fullhelms and ablative helmets now properly hide your face.
tweak: Blackshield now gets normal helmets on their armor vouchers again. Full helmets can be crafted through the crafting menu.
/:cl:

All feedback is welcome.

### CODE REVIEW

**Comment by ThatGuyFromThatPlace** [comment]
I never managed to respond to you in general chat when you were talking about changing these, but... Maybe talk to the original spriter first

File: code/datums/craft/recipes/clothing.dm [hunk 1]
```diff
@@ -263,6 +263,17 @@
 		list(QUALITY_ADHESIVE, 15, 30)
 	)
 
+/datum/craft_recipe/clothing/fullhelm_bs
+	name = "blackshield full helmet"
+	result = /obj/item/clothing/head/helmet/ballistic/shieldfull
+	steps = list(
+		list(/obj/item/clothing/head/helmet/ballistic/militia, 1, "time" = 30),
+		list(CRAFT_MATERIAL, 2, MATERIAL_STEEL),
```
Comment (ThePainkiller) [inline code/datums/craft/recipes/clothing.dm:271]: Full helmets offer a bit more protection than the average helmet. I suggest adding at least 2 sheets of plasteel as well to the recipe to make sense balance-wise.

Comment (cdb-is-not-good) [reply code/datums/craft/recipes/clothing.dm:271]: Agree

**Comment by cdb-is-not-good** [comment]
While I'm not necessarily opposed to changing the sprites I do want to ask that you consult the original spriters when possible. It's a bit of a faux pax to modify someone else's work without at least talking to em

**Comment by Kaksisilma** [comment]
This was handled poorly and I forgot to speak with them out of excitement. That being said, I talked to Kavrick just now and I don't see the need to get this merged anymore after I was informed of the plans to change the rest of the BS sprites. Though I'm still not particularly fond of how the full helm interacts with the rest of their uniform currently, this ends up being largely a subjective issue. I'll go ahead and close this.

**Pull request closed without merging**